src/shared/src/backend-supports/change-request-supports.ts

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
since this function should only be called by the backend, it should go in the backend change request utils